docs/kubernetes.rst

```diff
@@ -0,0 +1,77 @@
+########################
+Using the Kubernetes API
+########################
+
+Safir clients are encouraged to use the `kubernetes-asyncio <https://github.com/tomplus/kubernetes_asyncio>`__ Python module.
+It provides an async API for Kubernetes that will work naturally with FastAPI applications.
+
+Most Kubernetes work can be done by calling that API directly, with no need for Safir wrapper functions.
+Safir provides a convenient :py:func:`~safir.kubernetes.initialize_kubernetes` function that chooses the correct way to load the Kubernetes configuration depending on whether the code is running from within or outside of a cluster, and a framework for mocking the Kubernetes API for tests.
```

Comment:
```suggestion
Safir provides a convenient `~safir.kubernetes.initialize_kubernetes` function that chooses the correct way to load the Kubernetes configuration depending on whether the code is running from within or outside of a cluster, and a framework for mocking the Kubernetes API for tests.
```

Wherever you've got `:py:*:` in the docs, I recommend just dropping the explicit role and instead using single backticks. In Safir's doc's `conf.py` we've got `default_role = "py:obj"`, which makes single backticks using the Python role.